app/src/main/res/layout/share_layout.xml

```diff
@@ -47,4 +53,33 @@
         android:onClick="onClickSahre"
```

Comment:
onClickShare*
?

Reply:
i don't see it on the code...
can you check if you are on the wrong changeset?